bigbluebutton-html5/imports/api/meetings/server/modifiers/addMeeting.js

```diff
@@ -81,10 +83,12 @@ export default function addMeeting(meeting) {
     setBy: 'temp',
   };
 
+  newMeeting.welcomeProp.welcomeMsg = newMeeting.welcomeProp.welcomeMsg.replace('event:', '');
```

Comment:
Did you try matching more of the context - like `href="event:`. I want to avoid the case where welcome message

```
Welcome to the conference, we hope you enjoy our event:
* point 1
...
```

gets matched and modified.